### PR TITLE
Removed futures from view

### PR DESCRIPTION
**Pull request by hervandevliert**

### CODE REVIEW

File: cargo-sample/src/test/scala/io/cafienne/bounded/cargosample/SpecConfig.scala [hunk 1]
```diff
@@ -109,7 +109,7 @@ object SpecConfig {
       |       publish-confirmations = on
       |       publish-plugin-commands = on
       |       journal {
-      |          plugin = "akka.persistence.journal.inmem"
+      |          plugin = "inmemory-journal"
```
Comment (olger) [inline cargo-sample/src/test/scala/io/cafienne/bounded/cargosample/SpecConfig.scala]: Why did you change the journal type ? 
There is something with the different journals that prevent it running specific kind of tests. I believe it had todo with creating events with a temporary actor and thereafter use those in the actor that is tested (aggregate root)

Comment (hervandevliert) [reply cargo-sample/src/test/scala/io/cafienne/bounded/cargosample/SpecConfig.scala]: Because of this error when running the test:

"An exception or error caused a run to abort: Unsupported read journal akka.persistence.journal.inmem, please switch to cassandra for production 
java.lang.RuntimeException: Unsupported read journal akka.persistence.journal.inmem, please switch to cassandra for production"

"akka.persistence.journal.inmem" is not supporte.
See io.cafienne.bounded.akka.persistence.ReadJournalProvider

Comment (olger) [reply cargo-sample/src/test/scala/io/cafienne/bounded/cargosample/SpecConfig.scala]: Hm, after checkout and running a clean with test, get the next error:

```
error] Failed tests:
[error] 	io.cafienne.bounded.cargosample.projections.CargoQueriesSpec
[error] (cargoSample / Test / test) sbt.TestsFailedException: Tests unsuccessful
```
It needs some additional fixing it seems.

Comment (olger) [reply cargo-sample/src/test/scala/io/cafienne/bounded/cargosample/SpecConfig.scala]: BTW: something went wrong with the Configuration based setup as the akka.inmem was required for certain kind of tests in the past. We'd better have a good look on what is the issue here and if we could simplify the whole thing.
